### ISSUE TITLE
Cannot publish snapshot

### ISSUE THREAD

**Issue by 3flex** [detekt/detekt#6707]
## Expected Behavior
Snapshots are published.

## Observed Behavior
```
Execution failed for task ':detekt-gradle-plugin:publishDetektBasePluginPluginMarkerMavenPublicationToSonatypeSnapshotRepository'.
> Failed to publish publication 'detektBasePluginPluginMarkerMaven' to repository 'sonatypeSnapshot'
   > Could not PUT 'https://oss.sonatype.org/content/repositories/snapshots/dev/detekt/gradle/base/dev.detekt.gradle.base.gradle.plugin/main-SNAPSHOT/maven-metadata.xml'. Received status code 403 from server: Forbidden
```

## Steps to Reproduce
Check the logs for Deploy Snapshot GitHub action: https://github.com/detekt/detekt/actions/runs/7166465620/job/19510549237#step:7:445

## Your Environment

* Version of detekt used:
* Version of Gradle used (if applicable):
* Gradle scan link (add `--scan` option when running the gradle task):
* Operating System and version:
* Link to your project (if it's a public repository):

**Comment by cortinico** [comment]
That happened because we start publishing on `dev.detekt.*` which we haven't verified yet. I've opened a ticket here https://issues.sonatype.org/browse/OSSRH-97547

**Comment by 3flex** [comment]
Oh right... We could change the coordinates back until that's approved?

**Comment by cortinico** [comment]
> Oh right... We could change the coordinates back until that's approved?

Yup we could 👍 Yet we'll need to do this anyway so I don't mind not having snapshots for some days

**Comment by 3flex** [comment]
Fixed by #6779
